src/App.tsx

```diff
@@ -16,80 +17,171 @@ export const goodsFromServer = [
 ];
 
 enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
+  NONE = 'n',
+  ALPHABET = 'a',
+  LENGTH = 'l',
```

Comment:
```suggestion
  NONE = 'none',
  ALPHABET = 'alphabet',
  LENGTH = 'length',
```
better not to use short names